api/v1alpha1/humiocluster_types.go

```diff
@@ -118,9 +118,6 @@ type HumioNodeSpec struct {
 	// DataVolumeSource is the volume that is mounted on the humio pods. This conflicts with DataVolumePersistentVolumeClaimSpecTemplate.
 	DataVolumeSource corev1.VolumeSource `json:"dataVolumeSource,omitempty"`
 
-	// AuthServiceAccountName is the name of the Kubernetes Service Account that will be attached to the auth container in the humio pod.
-	AuthServiceAccountName string `json:"authServiceAccountName,omitempty"`
-
```

Comment:
What happens to existing custom resources if we rip it out like this?
Depending on the answer to that, we may keep this around, but just mark it as deprecated/unused. Previously I've been keeping the old fields around in our struct's just so that the generated CRD yaml continues to recognize any potential fields that may be set where we then just ignore whatever the value is.

Reply:
Added this back into the spec.